grpc/src/main/java/com/linecorp/armeria/internal/server/grpc/AbstractServerCall.java

```diff
@@ -540,7 +540,14 @@ public static HttpHeaders statusToTrailers(
         GrpcTrailersUtil.addStatusMessageToTrailers(
                 trailersBuilder, status.getCode().value(), status.getDescription());
 
-        MetadataUtil.fillHeaders(metadata, trailersBuilder);
+        try {
+            MetadataUtil.fillHeaders(metadata, trailersBuilder);
+        } catch (Exception e) {
```

Comment:
ok.

Reply:
If I understood correctly, what @jrhee17 asked is to ensure `RequestLog.responseCause()` returns the exception we caught in this block when this happened. I think there's no clean way to propagate this exception at the moment because this method is currently called from different places and it'd be their responsibility to catch and handling them properly to achieve the idea.

Given the usage of custom metadata serializer is fairly rare and we already tell our users about their bad implementation, I'd suggest keeping this as it is in this PR and follow-up when we find a bad serializer implementation is more common problem than we thought.